src/main/java/nextstep/strategy/RandomLineStrategy.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.strategy;
+
+import java.util.Random;
+
+public class RandomLineStrategy implements LineStrategy {
+    private static final Random random = new Random();
```

Comment:
ThreadLocalRandom을 써보면 어떨까요?

Reply:
[Class ThreadLocalRandom by Docs of Oracle](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadLocalRandom.html)

> When applicable, use of ThreadLocalRandom rather than shared Random objects in concurrent programs will typically encounter much less overhead and contention.

그런데, Oracle 에서 이런 문제를 제시하고 있네요.

> Instances of ThreadLocalRandom are not cryptographically secure. Consider instead using [SecureRandom](https://docs.oracle.com/javase/8/docs/api/java/security/SecureRandom.html) in security-sensitive applications. Additionally, default-constructed instances do not use a cryptographically random seed unless the [system property](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getProperty-java.lang.String-) java.util.secureRandomSeed is set to true.

새로운 클래스들을 알아갑니다. 🙇